pkg/omf/omf.fish

```diff
@@ -55,9 +55,9 @@ function omf -d "Oh My Fish"
     case "c" "cd"
       switch (count $argv)
         case 1
-          omf.cd
+          omf.packages.cd
```

Comment:
So better pull that code from `omf.packages.cd` and bring up here, it's more explicit, and as `$OMF_PATH` isn't a plugin more concise too.